### PR TITLE
feat(sourcemaps): Add setup flow for sentry-cli

### PR DESCRIPTION
**Pull request by lforst**
Resolves https://github.com/getsentry/sentry-wizard/issues/312

Adds catch-all sourcemap flow that:
- Will install `@sentry/cli`
- Asks the user to provide a path to their build artifacts
- Asks the user to verify that their build tool is generating source maps
- Will create a `package.json` script `sentry:ci` that will inject debug IDs into build artifacts and upload the artifacts
- Create a `.sentryclirc` file with an auth token so that the user can try the whole flow locally

### CODE REVIEW

File: src/sourcemaps/tools/sentry-cli.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      alreadyInstalled: hasPackageInstalled('@sentry/cli', packageDotJson),
+    });
+
+    let validPath = false;
+    let relativeArtifactPath;
+    do {
+      const rawArtifactPath = await abortIfCancelled(
+        clack.text({
+          message:
+            'Provide a path to the folder that contains your build artifacts:',
```
Comment (Lms24) [inline src/sourcemaps/tools/sentry-cli.ts:31]: m: Should we rephrase this as a question? 

For example
```suggestion
            'Where are your build artifacts located?',
```

File: src/sourcemaps/tools/sentry-cli.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        )}`,
+        options: [{ label: 'I checked. Continue!', value: true }],
+        initialValue: true,
+      }),
+    );
+
+    packageDotJson.scripts = packageDotJson.scripts || {};
+    packageDotJson.scripts[
+      'sentry:ci'
+    ] = `sentry-cli sourcemaps inject --org ${options.orgSlug} --project ${options.projectSlug} ${relativePosixArtifactPath} && sentry-cli sourcemaps upload --org ${options.orgSlug} --project ${options.projectSlug} ${relativePosixArtifactPath}`;
```
Comment (Lms24) [inline src/sourcemaps/tools/sentry-cli.ts:81]: m: We need to pass the `url` here, too, for self-hosted, right?

File: src/sourcemaps/tools/sentry-cli.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            ),
+          },
+        ],
+        initialValue: true,
+      }),
+    );
+
+    if (!addedToCI) {
+      clack.log.info("Don't forget! :)");
+    }
```
Comment (Lms24) [inline src/sourcemaps/tools/sentry-cli.ts:117]: Nice, we might wanna add this to the Vite flow 😅

File: src/sourcemaps/tools/sentry-cli.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    });
+
+    let validPath = false;
+    let relativeArtifactPath;
+    do {
+      const rawArtifactPath = await abortIfCancelled(
+        clack.text({
+          message:
+            'Provide a path to the folder that contains your build artifacts:',
+          placeholder: `path${path.sep}to${path.sep}your${path.sep}build${path.sep}output`,
```
[the remark this answers is not in the document]
Comment (lforst) [reply src/sourcemaps/tools/sentry-cli.ts:32]: Not everybody might know what dist is, I'm going with `./out`.

File: src/sourcemaps/tools/sentry-cli.ts (hunk 2 above)
Comment (lforst) [reply src/sourcemaps/tools/sentry-cli.ts:81]: ah right

**Review by Lms24 — APPROVED** [review]
